### PR TITLE
fix: set iface correctly

### PR DESCRIPTION
**Pull request by xhdix**

### CODE REVIEW

File: utils/iface.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    from scapy.all import IFACES
+    print(IFACES)
+
+
+def get_iface_object(name_or_index):
+    from scapy.all import IFACES
+    if name_or_index == "":
+        print("please set a correct iface name or index number")
+        show_ifaces()
+        exit()
```
Comment (bassosimone) [inline utils/iface.py:13]: Does `exit()` cause the exit code to be `0`? In such a case, maybe I'd `exit(1)` to indicate there was an error.

**Review by bassosimone — COMMENTED** [review]
Works as intended for me on macOS:

* `./tracevis.py --dns` ✔️ 
* `./tracevis.py --show-ifaces` ✔️ 
* `./tracevis.py --iface 11 --dns` ✔️ 
* `./tracevis.py --iface en0 --dns` ✔️ 

I have a question about your usage of `exit()` though.

File: utils/iface.py (hunk 1 above)
Comment (xhdix) [reply utils/iface.py]: Thanks! done!

**Comment by xhdix** [comment]
Thank you @bassosimone ! :heart: :hibiscus:

**Pull request merged**